### DISCUSSION TITLE
https://blog.3place.cn/3place/lmyn/LMYN019.html

### DISCUSSION THREAD

**Discussion by giscus** [hbaolong/hbaolong.github.io#65] [General]
# https://blog.3place.cn/3place/lmyn/LMYN019.html

https://blog.3place.cn/3place/lmyn/LMYN019.html

**Comment by picklecai** [comment]
乐得身毛直竖

在我们的语境里，这是害怕或者愤怒。
在他们的语境里，这是极度欢乐。

情绪到了极致，在身体上的反应，可能就没有差别了。

**Comment by picklecai** [comment]
有两个勇猛灵巧的罗刹，
名叫摩哩遮〔194〕和苏波呼〔195〕；
他们俩对准了我的祭坛，
投掷了雨点般的血和肉。 （1.18.5）

勇猛灵巧。

**Comment by picklecai** [comment]
祭典那坚定的目的，
就这样受到了阻挡。
费了半天劲现在泄了气，
因此我就离开了那地方。 （1.18.6）

国王呀！
我没有想/让我的怒气爆发；
因为举行这样祭典，
不允许我发出咒骂。 （1.18.7）

不迁怒的众友。

**Comment by picklecai** [comment]
国王呀！请你不要
动感情疼爱儿子！
我敢向你做保证，
他会把罗刹杀死。 

触龙说赵太后。

**Comment by picklecai** [comment]
大牟尼的这一番话，
激动着国王的心肠。
他听完了话以后，
吓得内心直发慌。
从宝座上站起来，
这一位大地之王，
心里头七上八下，
浑身发抖心慌张。 （1.18.20）

不愧是父亲的反应。

**Comment by picklecai** [comment]
众友的游说次序：
1. 我遇到了困难，但我忍着怒气。
2.  你儿子可以帮我解决这个困难。
3. 我会保护他的安全。
4. 我会让他得到幸福。
5. 他会因为帮我解决这个困难而声名远扬。
6. 只有你儿子能帮我解决这个困难。
7. 你要相信你儿子的能力，不要舍不得。
8. 即使是你自己想要功名，也要允许儿子来帮我解决这个困难。
9. 如果你的大臣们全都同意，那就把儿子交给我。
10. 我会祝你幸福，但还是要你把儿子交给我，不要舍不得。

自己的好处：解决两个小罗刹，完成祭祀。
罗摩的好处：安全、幸福、名气。
父亲的好处：功果、名声、幸福。

父亲的顾虑：舍不得儿子。
父亲的“救济”途径：征求大臣们的同意。

所有这些，最后十车王还是不同意的话，那十车王就是不顾儿子和自己的幸福和前途的“坏人”，是不听别人劝告的“执拗的人”。

所以听完这些，十车王的反应很激烈：晕过去了。
